Sources/NIOHTTP1Server/main.swift

```diff
@@ -399,6 +399,8 @@ let bootstrap = ServerBootstrap(group: group)
     // Set the handlers that are applied to the accepted Channels
     .childChannelInitializer { channel in
         channel.pipeline.addHTTPServerHandlers().then {
+            channel.pipeline.add(handler: HTTPServerPipelineHandler())
```

Comment:
I think `addHTTPServerHandlers` should have some parameter of whether to insert `HTTPServerPipelineHandler` automatically

Reply:
Do you mind if we open an issue to track this work and I deal with it in another PR? Doing it well requires a fairly substantial rewrite of that code, and I don't want to make this PR handle two jobs.

Reply:
@Lukasa works for me

Reply:
Works for me too but maybe we should tag 1.1.0 before this PR so we can sort out possible APIs before we then tag 1.2.0 which will contain this PR? But you two decide, I’ll be offline for most of the day and off next week